### PR TITLE
home page slider

### PR DESCRIPTION
**Pull request by MateuszUrbanowicz**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -12,5 +12,8 @@ var elixir = require('laravel-elixir');
  */
 
 elixir(function(mix) {
-    mix.less('app.less');
+    mix.less([
+        'app.less',
+        'animate.less'
```
Comment (PSkierniewski) [inline gulpfile.js]: animate.less should be added as @import in app.less, modifying gulpfile is not required.

File: resources/assets/less/app.less [hunk 2]
```diff
@@ -150,7 +150,60 @@ body {
 .error-box .col-md-6 {
   margin-bottom : 30px
 }
+
 /* helper classes */
 .margin-zero-top {
   margin-top : 0;
 }
+
+/* slider */
```
Comment (PSkierniewski) [inline resources/assets/less/app.less]: Slider styles should be added as separate less file and @imported to app.less

File: resources/views/blocks/slider.blade.php [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+</div>
+
+@section('footerScripts')
+
+<script src="{{ asset('/js/jquery.sequence-min.js') }}"></script>
+
+<script>
+    $(document).ready(function() {
+
+        $(document).ready(function(){
```
Comment (PSkierniewski) [inline resources/views/blocks/slider.blade.php]: We should use one of the free themes available on the sequence website.

**Comment by AdrianSkierniewski** [comment]
@MateuszUrbanowicz We should use configurable upload path from gzero config

``` PHP
    //'upload'       => [
    //    'path'   => public_path('uploads'),
    //    'public' => asset('uploads')
    //],
```

**Pull request merged**